android/src/com/mapswithme/maps/Framework.java

```diff
@@ -61,6 +61,13 @@
     public boolean buildings;
   }
 
+  public static class RouteAltitudeLimits
+  {
+    public int minRouteAltitude;
```

Comment:
> Why do we need this at all? I don't see places where this class is used. 

I'll be used in Android code by @alexzatsepin . He's waiting this PR to use it. The same situation with iOS code.

> Moreover, it seems quite strange to me that the method named GenerateRouteAltitudeChart returns min and max altitudes of a route. min and max altitudes are properties of a current route, not a chart.

There're the following options:
1. Add a special method for getting RouteAltitudeLimits;
2. Add this info RoutingInfo and return it with nativeGetRouteFollowingInfo();
3. Return altitude limits in method which generates route altitude diagram.

As for 1 method for returning limits and for generating diagram could return different result (for two different routes) because the route could be deleted/rebuilt;

As for 2 there's one more disadvantage: RoutingInfo contains significant amount info about following along route;

The third variant (implemented now): the method (nativeGenerateRouteAltitudeChartBits) is developed to be used by client to get info for altitude diagram. So it's useful for clients to get the info together. Moreover if everything is returned in one method we can guarantee that this information is for the same route.

Probably it's worth renaming it to nativeGenerateRouteAltitudeInfo or something like that.